### DISCUSSION TITLE
[REST] Kyuubi Rest Api Metrics

### DISCUSSION THREAD

**Discussion by zwangsheng** [apache/kyuubi#4243] [Q&A] [answered]
In master branch, how should we check the availability of Kyuubi Rest Api and record whether each Api is responding properly.

**Comment by zwangsheng** [comment]
Friendly ping @turboFei @bowenliang123

**Comment by turboFei** [comment, marked as answer]
<img width="614" alt="image" src="https://user-images.githubusercontent.com/6757692/216566399-bde6dab2-faff-4b15-a4c8-805772f1a6f7.png">

https://github.com/apache/incubator-kyuubi/issues/3156

<img width="1616" alt="image" src="https://user-images.githubusercontent.com/6757692/216567143-c99322df-32c6-4358-ab0b-793937c80486.png">

https://github.com/apache/kyuubi/pull/3951
